frontend/classes_subtypes.txt

```diff
@@ -0,0 +1,26 @@
+# This file represents the subtype relationships tree
+# Each line denotes a directed edge in the tree. Each consists of two names, x and y, denoting that type
+# x is a subtype of type y
+#
+#
+#             ----------- object -----------
+#            /    /    /    /\    \    \    \
+#           /    /    /    /  \    \    \    \
+#          /    /    /    /    \    \    \    \
+#       float  str bytes list tuple dict set function
+#        /
+#      int
+#      /
+#    bool
+#
+
+bool int
+int float
```

Comment:
I wonder if it makes sense to follow more closely the type hierarchy described in the documentation: https://docs.python.org/3.5/reference/datamodel.html
and not have int as a subtype of float (for instance)